src/main/java/edu/harvard/iq/dataverse/DOIDataCiteServiceBean.java

```diff
@@ -105,11 +109,15 @@ public HashMap lookupMetadataFromIdentifier(String protocol, String authority, S
         return metadata;
     }
 
+    @Override
     public String getIdentifierForLookup(String protocol, String authority, String separator, String identifier) {
+        logger.log(Level.FINE,"getIdentifierForLookup");
```

Comment:
This method seems the same to me for both EZId and Datacite and I suppose it should become the same for handlenet as well. If true we can pull this one up to the abstract service bean. Any future exception to the rule can still override.